.github/workflows/test_staging.yml

```diff
@@ -59,6 +59,11 @@ jobs:
         with:
           ssm_parameter: "STG_TWILIO_TEST_WORKSPACE_SID"
           env_variable_name: "TWILIO_WORKSPACE_SID"
+      - name: Set operating information file key
```

Comment:
@alebarquin is this the temporary practice environment?  If so, better to label this something similar to development rather than staging, so we don't confuse it with other staging accounts.